### PR TITLE
Release 1.10.0-beta1

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

Prepares `1.10.0-beta1` release 🚀

This version expose web-view tracking apis.

### Review checklist

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference
- [x] Add CHANGELOG entry for user facing change.

### CODE REVIEW

File: docs/log_collection.md [hunk 1]
```diff
@@ -214,9 +214,9 @@ The SDK changes its behavior according to the new value. For example, if the cur
 - if changed to `.granted`, the SDK will send all current and future data to Datadog;
 - if changed to `.notGranted`, the SDK will wipe all current data and will not collect any future data.
 
-Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your application sandbox.
+Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox](6), which can't be read by any other app installed on the device.
```
Comment (alai97) [inline docs/log_collection.md:217]:
```suggestion
Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox][6], which can't be read by any other app installed on the device.
```

File: docs/rum_collection/data_collected.md [hunk 2]
```diff
@@ -220,11 +220,12 @@ Network errors include information about failing HTTP requests. The following fa
 
 ## Data Storage
 
-Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your application sandbox.
+Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox](3), which can't be read by any other app installed on the device.
```
Comment (alai97) [inline docs/rum_collection/data_collected.md:223]:
```suggestion
Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox][3], which can't be read by any other app installed on the device.
```

File: docs/trace_collection.md [hunk 3]
```diff
@@ -205,14 +205,14 @@ Use one of the following values for `trackingConsent`:
...
-The SDK changes its behavior according to the new value. 
+The SDK changes its behavior according to the new value.
 
 For example, if the current tracking consent is `.pending`:
 
 - If changed to `.granted`, the SDK sends all current and future data to Datadog.
 - If changed to `.notGranted`, the SDK wipes all current data and does not collect future data.
 
-Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your application sandbox.
+Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox](11), which can't be read by any other app installed on the device.
```
Comment (alai97) [inline docs/trace_collection.md:215]:
```suggestion
Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox][11], which can't be read by any other app installed on the device.
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Numbered links need to be in brackets [].

**Review by alai97 — APPROVED** [review]
Looks good for docs!

**Review by xgouchet — APPROVED** [review]

**Pull request merged**